languages/ruby/exercises/concept/booleans/.docs/after.md

```diff
@@ -22,11 +22,7 @@
   end
   ```
 
-- It is common to use _truthy_ and _falsey_ values to determine the outcome of [conditional statements][control-expressions].
-
-  - `if`…`else` exists as a construct, similar to the [_C-family_ of programming languages][c-family]
-  - it is often more idiomatic to use `if` and `unless` as [expression modifiers to "guard" an expression][if-modifier]
-
+### Sample Usage 
   ```ruby
   1 + 1 if truthy
```

Comment:
[Lines 26-38](https://github.com/exercism/v3/blame/83b6a40174d0db1ff3fb4681c00eb860a8be1219/languages/ruby/exercises/concept/booleans/.docs/after.md#L26-L38) there are remaining references to if (and unless)

Reply:
@neenjaw Should be ok now :thinking: